app/scripts/components/common/mapbox/layer-legend.tsx

```diff
@@ -252,17 +251,11 @@ export function LayerLegend(
       )}
       renderBody={() => (
         <LegendBody>
-          <ShadowScrollbar
-            scrollbarsProps={{
-              autoHeight: true,
-              autoHeightMin: 32,
-              autoHeightMax: 120
-            }}
-          >
+
```

Comment:
Text is cut off here:

![Screenshot 2023-10-05 at 14 38 07](https://github.com/NASA-IMPACT/veda-ui/assets/1583415/f971c521-3496-4c23-873c-e121fc841609)

On `data-catalog/sedac-popdensity-yeargrid5yr-v4.11`